graph.go

```diff
@@ -28,6 +36,14 @@ func NewGraph(title, tmpl string) Graph {
 		ScvgSys:      []graphPoints{},
 		ScvgReleased: []graphPoints{},
 		ScvgConsumed: []graphPoints{},
+		STWSclock:    []graphPoints{},
+		MASclock:     []graphPoints{},
+		STWMclock:    []graphPoints{},
+		STWScpu:      []graphPoints{},
+		MASAssistcpu: []graphPoints{},
+		MASBGcpu:     []graphPoints{},
+		MASIdlecpu:   []graphPoints{},
+		STWMcpu:      []graphPoints{},
```

Comment:
you should be able to delete all of these, there is no difference between an zero slice and a slice with zero entries